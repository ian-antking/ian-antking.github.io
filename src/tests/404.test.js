import React from 'react';
import { render } from '@testing-library/react';
import Page404 from '../pages/404';

describe('404 Page', () => {
  it('renders the correct snapshot', () => {
    const { asFragment } = render(<Page404 />);

    expect(asFragment()).toMatchSnapshot();
  });
});
