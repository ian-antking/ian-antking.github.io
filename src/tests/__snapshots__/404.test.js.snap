// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 Page renders the correct snapshot 1`] = `
<DocumentFragment>
  <svg
    class="sc-AxjAm eSIdJy"
  >
    undraw_page_not_found_su7k.svg
  </svg>
  <h2>
    Oops, looks like you took a wrong turn.
  </h2>
</DocumentFragment>
`;
